apps/editor/src/App.tsx

```diff
@@ -16,6 +16,7 @@ import { SlashMenuPlugin } from "prosemirror-slash-menu";
 import {
   promptCommands,
   ProsemirrorSuggestcatPluginReact,
+  slashOpeningCondition,
```

Comment:
prior this we need to merge your PR in `prosemirror-suggestcat-plugin-react` and update the dependency here
is that correct?
if so I will merge this PR when everything is ready
thanks, no need for further actions from you here! 👍 🔥

Reply:
Correct, we need to publish that first. should have mentioned that